test/components/connect.spec.js

```diff
@@ -203,8 +202,8 @@ describe('React', () => {
 
       const pending = TestUtils.findRenderedComponentWithType(container, Container)
       const startedAt = pending.state.startedAts.testFetch
-      expect(startedAt.getTime()).toBeLessThan(new Date().getTime())
```

Comment:
I've had this test fail locally with just `npm test` a few times like this. Were you seeing it consistently?